subsys/bluetooth/audio/bap_scan_delegator.c

```diff
@@ -1338,7 +1338,7 @@ int bt_bap_scan_delegator_mod_src(const struct bt_bap_scan_delegator_mod_src_par
 	/* Verify that the BIS sync values is acceptable for the receive state */
 	for (uint8_t i = 0U; i < state->num_subgroups; i++) {
 		const uint32_t bis_sync = param->subgroups[i].bis_sync;
-		const uint32_t bis_sync_requested = internal_state->requested_bis_sync[i];
+		const uint32_t bis_sync_requested = internal_state->requested_bis_sync[i] << 1;
```

Comment:
I think it would be better if we shifted the `requested_bis_sync` when storing it, so that the `requested_bis_sync` and `param->subgroups[i].bis_sync;` are stored the same way

That would also remove the need to shift in `bt_bap_scan_delegator_set_bis_sync_state`